packages/template-typescript-minimal/package.json

```diff
@@ -4,19 +4,22 @@
   "private": true,
   "scripts": {
     "build": "pwa-kit-dev build",
-    "format": "pwa-kit-dev format \"**/*.{js,jsx,tsx}\"",
-    "lint": "pwa-kit-dev lint \"**/*.{js,jsx,tsx}\"",
-    "lint:fix": "npm run lint -- --fix",
+    "format": "pwa-kit-dev format \"**/*.{js,jsx,ts,tsx}\"",
+    "lint": "npm run lint:js && npm run typecheck",
+    "lint:fix": "npm run lint:js -- --fix",
+    "lint:js": "pwa-kit-dev lint \"**/*.{js,jsx,ts,tsx}\"",
     "push": "npm run build && pwa-kit-dev push",
     "save-credentials": "pwa-kit-dev save-credentials",
     "start": "pwa-kit-dev start",
     "start:inspect": "npm run start -- --inspect",
     "tail-logs": "pwa-kit-dev tail-logs",
-    "test": "pwa-kit-dev test"
+    "test": "pwa-kit-dev test",
+    "typecheck": "tsc --noEmit"
```

Comment:
Turns out we were never actually checking the types in our TypeScript project. Oops!